aws/acm-certificate/outputs.tf

```diff
@@ -1,21 +1,21 @@
 output "certificate_arn" {
   description = "The ARN of the certificate"
-  value       = element(concat(aws_acm_certificate.cert.*.arn, [""]), 0)
+  value       = aws_acm_certificate.cert[0].arn
 }
 
 output "acm_certificate_domain_validation_options" {
   description = "A list of attributes to feed into other resources to complete certificate validation. Can have more than one element, e.g. if SANs are defined. Only set if DNS-validation was used."
-  value       = flatten(aws_acm_certificate.cert.*.domain_validation_options)
+  value       = aws_acm_certificate.cert[0].domain_validation_options
```

Comment:
Why is this the same?

Reply:
There is only 0 or 1 elements in the list depending on a count with feature that can be 1 or 0. 

As there can be max 1 certificate created there is max one list returned. Added try to supply an empty list if count == 0.